examples/CMakeLists.txt

```diff
@@ -5,7 +5,9 @@ project(examples)
 option(WITH_IMGUI "Build ImGui based examples?" OFF)
 
 # Dependecies that can be accessed by any example
-add_subdirectory(deps/imgui)
+if (WITH_IMGUI)
```

Comment:
Can we add this in the same if with aditof-demo-imgui?

Reply:
I would keep it here. I wanted to separate the dependencies that can be shared between the examples from the examples.